File: wesafe.net/WeSafe.DAL.Entities/UnhandledException.cs
using System;

namespace WeSafe.DAL.Entities
{
    public class UnhandledException
    {
        public int Id { get; set; }

        public string UserId { get; set; }

        public virtual User User { get; set; }

        public string ErrorMessage { get; set; }

        public string StackTrace { get; set; }

        public DateTime DateTime { get; set; }
    }
}

File: wesafe.net/WeSafe.Web/ClientApp/src/app/models/unhandled-exception.ts
export interface IUnhandledException {
    userName: string;
    errorMessage: string;
    stackTrace: number;
    dateTime: string;
}

export class UnhandledException implements IUnhandledException {
    userName: string;
    errorMessage: string;
    stackTrace: number;
    dateTime: string;
}

export interface IUnhandledExceptionFilter {
    userName: string;
    fromDate: string;
    toDate: string;
    searchText: string;
}

export class UnhandledExceptionFilter implements IUnhandledExceptionFilter {
    userName: string;
    fromDate: string;
    toDate: string;
    searchText: string;
}
